src/NServiceBus.Core/DelayedDelivery/DelayedDeliveryFeature.cs

```diff
@@ -12,7 +12,10 @@ public DelayedDeliveryFeature()
             DependsOnOptionally<TimeoutManager>();
             Defaults(s =>
             {
-                var timeoutManagerAddressConfiguration = new TimeoutManagerAddressConfiguration(s.GetConfigSection<UnicastBusConfig>()?.TimeoutManagerAddress);
+                var timeoutManagerAddressConfiguration =
+                    new TimeoutManagerAddressConfiguration(
+                        s.GetConfigSection<UnicastBusConfig>()?.TimeoutManagerAddress
```

Comment:
This means that if both the config value and the code-first value have been set, the config value will take precedence. Is this what we want? Is this consistent with the other settings that have both config and code-first options?